### DISCUSSION TITLE
Changes to meeting time revert back to last year's

### DISCUSSION THREAD

**Discussion by stepstra** [sircharlo/meeting-media-manager#2359] [General]
Anyone got an idea why that is?

**Comment by stepstra** [comment]
@mtdvlpr Any idea? Don't know how to contact you directly ...

**Reply by e-Feitosa** [reply]
@mtdvlpr, I have the same problem, however, I edited the file that is in the cloud ForcedPrefs.json. This issue occurs because this day and time change to meetings is not saving directly to the cloud.

**Comment by mtdvlpr** [comment]
@stepstra I'd need more information. Your prefs, a video or reproduction steps. Are you using a congregation sync server?

**Reply by stepstra** [reply]
https://github.com/sircharlo/meeting-media-manager/assets/10519895/48a6c489-0efe-473f-91cb-709ec4cadfaf

**Reply by stepstra** [reply]
@mtdvlpr Any other info you need? We could do a screenshare sometime if you like.

**Reply by mtdvlpr** [reply]
@stepstra no  I see what you mean. I'll take a look at it when I have some spare time.

**Reply by stepstra** [reply]
@mtdvlpr I see, you have as much spare time as myself ... ;-)

**Reply by mtdvlpr** [reply]
Hi @stepstra, I can't seem to reproduce this. Could you describe every step you take from starting M3 until the bug occurs? Or maybe a screenshare session?

**Reply by stepstra** [reply]
@mtdvlpr Actually, I do nothing special. Start M3, set the new time, back to homepage, then back to settings and then it's reverted.

Yes, screenshare would be nice. I'll be home from 13:00 onwards. What's your preferred method? Teamviewer, Anydesk, Microsoft remote ...

**Reply by mtdvlpr** [reply]
I have TeamViewer installed. Which timezone are you in?

**Reply by stepstra** [reply]
If you live in the Netherlands it's the same, as I live in Germany. ;-)

**Reply by mtdvlpr** [reply]
@stepstra, is 16:00 okay for you?

**Reply by stepstra** [reply]
Sorry, can we try again tomorrow?

**Reply by mtdvlpr** [reply]
Yeah sure. Tomorrow at 14:00?

**Reply by mtdvlpr** [reply]
@stepstra, 14:00?

**Reply by stepstra** [reply]
Sorry, won't be home before 15:00

**Reply by mtdvlpr** [reply]
@stepstra, 15:30 then?

**Reply by stepstra** [reply]
Great

**Reply by stepstra** [reply]
@mtdvlpr Ready to go now. Just waiting for your session-code ...

**Reply by mtdvlpr** [reply]
@stepstra https://quicksupport.me/s123950757

**Comment by mtdvlpr** [comment]
For all interested parties: I found the cause of the bug (with help from @stepstra). It only goes wrong when you try to change the meeting time without changing the minutes (e.g. 13:00 to 10:00). A workaround is to change it to 10:30 first and then change it to 10:00. I'll release a fix for this as soon as I tackled the bug!

**Reply by mtdvlpr** [reply]
The fix has been released in v24.2.0
